src/main/resources/db/changesets/20231129_snapshotrequests.yaml

```
@@ -0,0 +1,60 @@
+databaseChangeLog:
+  - changeSet:
+      id: snapshot_requests
+      author: rjohanek
+      changes:
+        - createTable:
+            tableName: snapshot_requests
```

In general we use singular names for tables
```suggestion
            tableName: snapshot_request
```